elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -64,28 +69,49 @@
     @Autowired(required = false)
     private MeterRegistry meterRegistry;
 
+    @Value("${spring.dialect}")
```

Comment:
instead of asking user to put new/duplicate `dialect` property can we use from the JPAProperties?

Reply:
The dialects are different classes though.  I think the spring dialect is a JPA dialect, whereas our default dialect is an Elide concept.  I'm not sure we can do this.

Reply:
added default-dialect property to elide.aggregation-store